cmd/zstreamdump/zstreamdump.c

```diff
@@ -683,6 +687,7 @@ main(int argc, char *argv[])
 				print_block(buf,
 				    P2ROUNDUP(drrwe->drr_psize, 8));
 			}
+			payload_size = P2ROUNDUP(drrwe->drr_psize, 8);
```

Comment:
If you feel like cleaning this up a bit, we could avoid repeating the P2ROUNDUP 3x:
```
payload_size = ...;
ssread(... payload_size ...);
if (dump)
  print_block (... payload_size)
```